modules/directives/src/main/scala/scala/build/preprocessing/directives/ScriptWrapper.scala

```diff
@@ -0,0 +1,37 @@
+package scala.build.preprocessing.directives
+
+import scala.build.directives.*
+import scala.build.errors.BuildException
+import scala.build.options.{BuildOptions, ScriptOptions}
+import scala.cli.commands.SpecificationLevel
+
+@DirectiveExamples("//> using objectWrapper")
+@DirectiveExamples("//> using delayedInitWrapper")
+@DirectiveUsage(
+  "//> using delayedInitWrapper",
+  """`//> using objectWrapper`
+    |
+    |`//> using delayedInitWrapper`
+    |""".stripMargin
+)
+@DirectiveLevel(SpecificationLevel.RESTRICTED)
+@DirectiveDescription("Set parameter for the code wrapper for scripts")
+final case class ScriptWrapper(
+  @DirectiveName("object.wrapper")
+  objectWrapper: Boolean = false,
+  @DirectiveName("delayedInit.wrapper")
```

Comment:
Similarly here. Also, I believe we were to avoid camel case in directives' keys,
so
```suggestion
  @DirectiveName("wrapper.delayed.init")
```
or
```suggestion
  @DirectiveName("wrapper.delayed-init")
```